src/m365/spo/commands/file/file-roleinheritance-reset.ts

```diff
@@ -96,7 +96,7 @@ class SpoFileRoleInheritanceResetCommand extends SpoCommand {
         const requestOptions: AxiosRequestConfig = {
           url: `${args.options.webUrl}/_api/web/GetFileByServerRelativeUrl('${formatting.encodeQueryParameter(fileURL)}')/ListItemAllFields/resetroleinheritance`,
           headers: {
-            accept: 'application/json;odata.metadata=none'
+            accept: 'application/json'
```

Comment:
@nanddeepn sorry there was a last minute change in the issue 😜. could we align ? also in the next command 🙏

```suggestion
            accept: 'application/json;odata=nometadata'
```